artman/config/proto/user_config.proto

```diff
@@ -16,10 +16,12 @@ syntax = "proto3";
 
 package googleapis.artman;
 
-// Artman user config is used to store user settings. All configuration are
-// optional unless being explicitly marked required. Having some configuration
-// will simplify the CLI invocation (e.g. github config), or change the
-// execution behavior (e.g. local.toolkit).
+// The Artman user config is used to store user settings. All configuration is
+// optional unless it is explicitly marked as required. Configuration can
+// simplify the CLI invocation (e.g. github config), or change the behavior.
+// For example, when `local.toolkit` is configured to a local toolkit path, it
```

Comment:
suggest: "For example, if `local.toolkit`is specified, artman will run toolkit from a local source repository when executed in local mode."